docs/001_develop/03_client-capabilities/014_utility-methods/11_serializer-util.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Serializer utilities'
+sidebar_label: 'Serializer utilities'
+id: serializer-util
+keywords: [utils, utility, serializer, serialization, deserialization, complex, robust]
+tags:
+  - utils
+  - utility
+  - serializer
+  - serialization
+  - deserialization
+  - complex
+  - robust
```

Comment:
Let's get some more keywords in here relating to each of the individual functions, especially to JSON etc. This is a useful utility that people ask about a lot